jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/handler/ContextHandler.java

```diff
@@ -666,6 +666,13 @@ protected void notifyExitScope(Request request)
         }
     }
 
+    private static <T> List<T> reverse(List<T> list)
```

Comment:
This should be extracted out in a helper class, as this implementation is duplicated, a bit expensive and could easily be optimized later on if deemed necessary.

Reply:
Done.